### ISSUE TITLE
angle loss

### ISSUE THREAD

**Issue by DayBreak-u** [ailab26/pfld-lite#2]
hello,how use the angel loss,Is it necessary to add auxiliary network of Angle prediction？

**Comment by xindongzhang** [comment]
Thanks for your feedback, I will soon upload the full version of PFLD. In my experiments, it is not necessary to add auxiliary network. But from theoretical perspective, adding auxiliary network to weight the importance of the batch training can ease the data imbalance in facial landmark detection.

**Comment by xindongzhang** [comment]
@ouyanghuiyu FYI

**Comment by DayBreak-u** [comment]
thanks! convert mxnet to caffe is easy,why try it !  In that case, you can use residuals！

**Comment by xindongzhang** [comment]
> thanks! convert mxnet to caffe is easy,why try it ! In that case, you can use residuals！

Yep, caffe seems like a good alternative to ONNX representation, thanks for your advice.

**Comment by DayBreak-u** [comment]
Convert to onnx is due to use mnn? caffe2mnn is available

**Comment by xindongzhang** [comment]
> Convert to onnx is due to use mnn? caffe2mnn is available

Honestly, I just wanna get through the onnx-mnn pipeline, but caffe-mnn、pb-mnn seem more stable than onnx-mnn when deploying models to edge-devices.
